meson_options.txt

```diff
@@ -2,3 +2,9 @@ option('preload_libasan',
        type: 'string',
        value: 'no',
        description: 'Path to libasan.so to preload')
+
+
+option('default_pkcs11_module',
+    type : 'string',
+    value : 'no',
```

Comment:
This should probably be an empty string and caught in meson.build, unless we really want to allow not setting any default (that would require C code changes too).